ecosystem/sep-0012.md

```diff
@@ -140,21 +140,17 @@ returned in the response instead.
 
 Different types of customers may have different KYC requirements depending on the action a given customer wants to take.
 The `type` parameter is used to specify the action a customer wants to make so the server can identify what information
-must be collected. SEP-12 does not define what `type` values are accepted and instead leaves that to the other
-protocols, such as [SEP-6](sep-0006.md) and [SEP-31](sep-0031.md), that use SEP-12 for the actions associated with those
-protocols.
+must be collected.
 
 For example, if a customer is being KYC'd as a SEP-31 sender, they may only require full name and email, but a SEP-31
-receiver would require banking information in order to receive a direct deposit. The `type` parameter could also be used
-to indicate that a customer is an organization instead of a person.
+receiver would require banking information in order to receive a direct deposit.
 
 Note that it is possible for the same customer to have different `status` values for different `type` parameters. For
-example, a customer could have an `ACCEPTED` status for the `small-transaction-amount` `type` parameter but have a
-`NEEDS_INFO` status for the `large-transaction-amount` type. Therefore it is recommended to always pass the `type`
-parameter when making requests to `GET /customer` or `PUT /customer`, even though the field is optional to accomodate
-for implementations that do not require `type`.
+example, a customer could have an `ACCEPTED` status for the `seo31-sender` `type` parameter but have a `NEEDS_INFO`
+status for the `sep31-receiver` type.
 
-If the implementor requires the same set of fields for all customers, there is no need for the `type` parameter.
+Currently, the use of the `type` parameter is limited to SEP-31 only, supporting only two types: `sep31-sender` and
```

Comment:
This is only true for the anchor platform. We can't change the previous descriptions as there could be different implementations (e.g. Polaris) that define their own types.

Reply:
Will revert it back